app/controllers/mission_control/jobs/jobs_controller.rb

```diff
@@ -36,4 +38,12 @@ def filtered_jobs
     def jobs_status
       params[:status].presence&.inquiry
     end
+
+    def set_backtrace_cleaner
+      @backtrace_cleaner = @application.servers[server_id]&.backtrace_cleaner
```

Comment:
You don't need to find the server here, [it's set for you in the `ApplicationScoped` concern](https://github.com/rails/mission_control-jobs/blob/7f71ec89b6213361e29dbc985495b5b8f473d995/app/controllers/concerns/mission_control/jobs/application_scoped.rb#L26-L36) 😊 So you can replace this by `@server`.

Reply:
Thank you. Clearly I missed that in my pursuing around the code base.